src/aap_eda/api/views/decision_environment.py

```diff
@@ -41,6 +42,11 @@
 
 class CreateDecisionEnvironmentMixin(CreateModelMixin):
     def create(self, request, *args, **kwargs):
+        # Restrict creation to superusers
+        if not request.user.is_superuser:
+            raise PermissionDenied(
+                "Only superusers can create a Decision Environment."
+            )
```

Comment:
What return code is being returned here?

Can we add unit test(s) to ensure we're getting the proper return code and to ensure this functionality works as expected with and without super_user permissions